Documentation/operations/upgrade.rst

```diff
@@ -602,6 +626,10 @@ Full list of updated Helm values:
 +----------------------------------------------+--------------------------------------------+
 | global.ipvlan.masterDevice                   | ipvlan.masterDevice                        |
 +----------------------------------------------+--------------------------------------------+
+| global.k8sServiceHost                        | k8sServiceHost                             |
++----------------------------------------------+--------------------------------------------+
+| global.k8sServicePort                        | k8sServicePort                             |
++----------------------------------------------+--------------------------------------------+
```

Comment:
Another one is (and the only reason I checked this was because it was on the original install options I used) is `config.blacklistConflictingRoutes=false` but felt too much for the example so I left it out...

Reply:
blacklistConflictingRoutes was deprecated in 1.9, so no need to include that one.